### ISSUE TITLE
Entry point pattern doesn't match entrypoints:with.dots

### ISSUE THREAD

**Issue by dimazest** [conda/conda-build#251]
Hi,

Could you please have a look to this. I define entry point as

``` python
entry_points={
    'console_scripts': [
        'poultry = poultry.main:dispatcher.dispatch',
    ],
},
```

https://github.com/dimazest/poultry/blob/e0f39277f0a219a4d3cd461b69ce5dd4422fc9dd/setup.py#L62

which entry point pattern at  https://github.com/conda/conda-build/blob/master/conda_build/scripts.py#L27 doesn't match.

Thanks.

**Comment by faph** [comment]
Conda build does not support `console_scripts` vs `gui_scripts` as far as I can see it (unfortunately). `entry_points` in `meta.yml` should be a simple list.

**Comment by asmeurer** [comment]
For gui_scripts, the only option is to set [osx_is_app](http://conda.pydata.org/docs/building/meta-yaml.html#python-app), which makes all entry points "gui_scripts" on OS X.

**Comment by frol** [comment]
I'd like to let you know that I have implemented a pull-request #690 fixing this issue.
